bin/binarize.sh

```diff
@@ -280,5 +289,16 @@ do
    fi
 done
 
+# In de future, if we want to inlude the user overlay workspace to all our binary images, we could also be
```

Comment:
In "the"

Reply:
"include"

Reply:
Thanks! https://github.com/shadow-robot/sr-build-tools/pull/709/commits/4fc7dade99c6ab414af9d524dada88bbcc4a7631